backend/repository/config.go

```diff
@@ -56,50 +56,19 @@ type ThriftServiceMap map[string]map[string]*ThriftService
 
 // EndpointConfig stores configuration for an endpoint.
 type EndpointConfig struct {
-	ID               string                    `json:"endpointId"`
-	Type             ProtocolType              `json:"endpointType"`
-	HandleID         string                    `json:"handleId"`
-	ThriftFile       string                    `json:"thriftFile"`
-	ThriftFileSha    string                    `json:"thriftFileSha,omitempty"`
-	ThriftMethodName string                    `json:"thriftMethodName"`
-	WorkflowType     string                    `json:"workflowType"`
-	ClientID         string                    `json:"clientID"`
-	ClientMethod     string                    `json:"clientMethod"`
-	TestFixtures     []*TestStub               `json:"testFixtures"`
-	Middlewares      []*codegen.MiddlewareSpec `json:"middlewares"`
-	ReqHeaderMap     map[string]string         `json:"reqHeaderMap"`
-	ResHeaderMap     map[string]string         `json:"resHeaderMap"`
-}
-
-// TestStub saves stubbed requests/responses for an endpoint test.
-type TestStub struct {
-	TestName               string                 `json:"testName"`
-	EndpointID             string                 `json:"endpointId"`
-	HandlerID              string                 `json:"handlerId"`
-	EndpointRequest        map[string]interface{} `json:"endpointRequest"`
-	EndpointRequestString  string                 `json:"endpointRequestString"`
-	EndpointReqHeaders     map[string]string      `json:"endpointReqHeaders"`
-	EndpointReqHeaderKeys  []string               `json:"endpointReqHeaderKeys"`
-	EndpointResponse       map[string]interface{} `json:"endpointResponse"`
-	EndpointResponseString string                 `json:"endpointResponseString"`
-	EndpointResHeaders     map[string]string      `json:"endpointResHeaders"`
-	EndpointResHeaderKeys  []string               `json:"endpointResHeaderKeys"`
-	ClientStubs            []ClientStub           `json:"clientStubs"`
-	TestServiceName        string                 `json:"testServiceName"`
-}
-
-// ClientStub saves stubbed client request/response for an endpoint test.
-type ClientStub struct {
-	ClientID             string                 `json:"clientId"`
-	ClientMethod         string                 `json:"clientMethod"`
-	ClientRequest        map[string]interface{} `json:"clientRequest"`
-	ClientRequestString  string                 `json:"clientRequestString"`
-	ClientReqHeaders     map[string]string      `json:"clientReqHeaders"`
-	ClientReqHeaderKeys  []string               `json:"clientReqHeaderKeys"`
-	ClientResponse       map[string]interface{} `json:"clientResponse"`
-	ClientResponseString string                 `json:"clientResponseString"`
-	ClientResHeaders     map[string]string      `json:"clientResHeaders"`
-	ClientResHeaderKeys  []string               `json:"clientResHeaderKeys"`
+	ID               string                       `json:"endpointID"`
```

Comment:
This is quite a breaking change that we should not make, it requires zanzibar users to change their JSON configs for no good reason. `id` is conventional in JSON, so does the camelcase of `endpointId`.  And `ID` is mandated by golint.

Reply:
good reason is `ID` and `Id` , `handler` and `handle` are inconsistent across repo, as long as we have consistent naming in the repo I don't mind `Id` or `ID`

Reply:
`endpointId` is in JSON config while `ID` is in Go code. As long as it's all `somethingId` in JSON and `somethingID` in Go, it's consistent.